connectivity/FEATURE_BLE/libraries/cordio_stack/ble-host/sources/hci/dual_chip/hci_evt.c

```diff
@@ -217,6 +217,7 @@ static const hciEvtParse_t hciEvtParseFcnTbl[] =
   hciEvtParseLeConnCteReqEnableCmdCmpl,
   hciEvtParseLeConnCteRspEnableCmdCmpl,
   hciEvtParseLeReadAntennaInfoCmdCmpl,
+#if 0
```

Comment:
there's a #define that you can use that selects the ble version 5.1 which compiles cis/bis etc. out

Reply:
I haven't seen any in the host.

Reply:
if this is landing in mbed-os then we should make one

I guess you could change the size of the array to avoid the nulls

Reply:
I improved this in the last commit.